pkg/option/config.go

```diff
@@ -3362,6 +3385,15 @@ func (c *DaemonConfig) Populate(vp *viper.Viper) {
 		c.LogOpt = m
 	}
 
+	c.BPFEventsMapRateLimit = vp.GetUint32(BPFEventsMapRateLimit)
+	c.BPFEventsMapBurstLimit = vp.GetUint32(BPFEventsMapBurstLimit)
+	if c.BPFEventsMapRateLimit > 0 && c.BPFEventsMapBurstLimit == 0 {
+		c.BPFEventsMapBurstLimit = 3 * c.BPFEventsMapRateLimit
+	}
+	if c.BPFEventsMapBurstLimit > 0 && c.BPFEventsMapRateLimit == 0 {
+		c.BPFEventsMapRateLimit = c.BPFEventsMapBurstLimit
```

Comment:
documentation says that specifying the `BPFEventsMapRateLimit` of `0` will disable the rate limiting feature. Yet you are copying burst value when it equals 0. I believe that for clarity it should be kept `0` (disabled) even when non-zero burst is provided.

Reply:
I'd prefer the behaviour when we're setting rate limit when burst limit option is present instead of simply ignoring burst limit when rate limit is not defined. Burst limit without rate limit is a valid scenario - user might only be interested in rate limiter not allowing more than X events per second at any time, where X is the burst limit. It might be confusing for some if we force users to define some random rate limit to have rate limiting enabled. This branch detects that scenario and sets rate limit to a value that won't have impact (if rate limit = burst limit, all rate limiter tokens are refreshed every second).

That being said, the documentation should be corrected to say that it's allowed.

Reply:
I'm fine with that if behavior is consistent with the docs.